app/views/funding_form/check_answers.html.erb

```diff
@@ -147,10 +147,12 @@
             name: "additional_comments"
           } %>
 
-          <h2 class="govuk-heading-m"><%= t('funding_form.check_your_answers.section_5.heading') %><h2>
-          <p class="govuk-body"><%= t('funding_form.check_your_answers.section_5.description') %></p>
+          <%= render "govuk_publishing_components/components/button", text: t("funding_form.check_your_answers.submit"), margin_bottom: true %>
 
-          <%= render "govuk_publishing_components/components/button", text: "Accept and send", margin_bottom: true %>
+          <%= render "govuk_publishing_components/components/inset_text", {
+            text: t("funding_form.check_your_answers.section_5.description", privacy_link: link_to(t("funding_form.check_your_answers.section_5.privacy_link_text"), "/government/publications/privacy-notice-for-grants-management-function-recipient-registration/privacy-notice-for-grants-management-function-recipient-registration", :target => "_blank")).html_safe,
```

Comment:
We may be able to rewrite this to not need `html_safe` on it – non-tested code below:
```
<%= render "govuk_publishing_components/components/inset_text", do %>
  <%= t(
    "funding_form.check_your_answers.section_5.description",
    privacy_link: link_to(
      t("funding_form.check_your_answers.section_5.privacy_link_text"),
      "/government/publications/privacy-notice-for-grants-management-function-recipient-registration/privacy-notice-for-grants-management-function-recipient-registration",
      target: "_blank",
      class: "govuk-link govuk-link--no-visited-state"
  ) %>
<% end %>
```

Reply:
Unfortunately I'm just seeing the HTML being rendered without `html_safe`.  But I've taken an opportunity to tidy the code and move the class to only the link.

Reply:
Cool, it may be required by the way yml&html work together.